### ISSUE TITLE
XmlValidator should search for schema in classpath

### ISSUE THREAD

**Issue by longtimeago** [teamed/qulice#273]
XmlValidator should search schema in classpath before trying to create URL and load from web
This example should pass validation if spring-beans-2.0.xsd can be located in classpath
```xml
<?xml version="1.0" encoding="UTF-8"?>
<beans xmlns="http://www.springframework.org/schema/beans"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       xmlns:util="http://www.springframework.org/schema/util"
       xsi:schemaLocation="http://www.springframework.org/schema/beans spring-beans-2.0.xsd">
</beans>
```

**Comment by krzyk** [comment]
@carlosmiranda jcabi/jcabi-xml#31 has been fixed, maybe we can continue here

**Comment by carlosmiranda** [comment]
@longtimeago @krzyk can't work on this yet, please see https://github.com/jcabi/jcabi-xml/issues/94.

**Comment by carlosmiranda** [comment]
@alex-palevsky waiting for #498

**Comment by carlosmiranda** [comment]
@davvd see PR #498.

**Comment by carlosmiranda** [comment]
@davvd waiting for #498.

**Comment by davvd** [comment]
> @davvd waiting for #498.

@carlosmiranda of course, let's wait for #498

**Comment by davvd** [comment]
@longtimeago #498 is closed, it was an impediment

**Comment by carlosmiranda** [comment]
@longtimeago See PR #607 - finally, after a few years... :)

**Comment by longtimeago** [comment]
@carlosmiranda nice :)

**Comment by carlosmiranda** [comment]
@longtimeago fixed in #607, may we close it now?

**Comment by longtimeago** [comment]
@carlosmiranda yep, finally! Thanks!

**Comment by krzyk** [comment]
@longtimeago it will be released in 0.15.2, probably in an hour

**Comment by davvd** [comment]
@carlosmiranda thanks, I just added **30 mins** to your account, payment `74570258`, 122 hours and 2 mins spent... added +30 to your rating, now it is equal to [+2586](http://www.netbout.com/b/35023?open=rating)
